src/components/Ranking/CityRankList.tsx

```diff
@@ -30,7 +30,12 @@ const CityRankList = ({ sido, isShow }: CityRankListProps) => {
       {isShow &&
         cityDustInfos &&
         cityDustInfos.map((city, cityIndex) => (
```

Comment:
```suggestion
        cityDustInfos?.map((city, cityIndex) => (
```

옵셔널을 활용한 코드가 더 깔끔해보여요